lib/hello_web/templates/hello/index.html.heex

```diff
@@ -0,0 +1,3 @@
+<section class="phx-hero">
+  <h2>Hello World, from Phoenix!</h2>
+</section>
```

Comment:
@YanethM missing an empty line at the end of the file. Tip: You can configure your code editor to automatically manages this for you